components/03-modules/slider/slider.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="slider {{ class }}" {{{ attributes }}}>
+    <div class="slider__handler">
+        <div class="slider__slides">
+            {{#each products }}
+                <div class="slider__item">
+                    <a href="{{ this.link }}" class="slider__link">
+                        {{ render '@image' this.image }}
+
+                        <div class="slider__item-handler">
+                            <h5 class="heading heading--fifth-level slider__name">
```

Comment:
By the way... in our `heading.scss` there is a line: 
```
h5 {
    @extend .heading--fifth-level;
}
```
I'm not sure, but is it necessary to use h5 with this class if h5 selector extends it anyway ?

Reply:
exterminated